conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/TezosTypes.scala

```diff
@@ -668,9 +674,9 @@ object TezosTypes {
   /** Utilities related to the baking process */
   object Baking {
 
-    //we'll move this to a proper chain configuration value later on
     /** how big is a baker roll in tez */
-    final val BakerRollsSize = BigDecimal.decimal(8000)
+    final val BakerRollsSize =
+      loadConfig[TezosConfiguration](namespace = "platforms").fold(_ => BigDecimal.decimal(8000), _.bakerRollsSize)
```

Comment:
you don't have to put default value here, it's enough when it is in the config(but check my previous comment where it should be)

Reply:
I don't see how else can I extract `right` value from `Either` without mapping `left` to a value of the `right`'s type